src/backend/distributed/executor/query_stats.c

```diff
@@ -0,0 +1,56 @@
+/*-------------------------------------------------------------------------
+ *
+ * query_stats.c
+ *    Statement-level statistics for distributed queries.
+ *
+ * Copyright (c) 2012-2018, Citus Data, Inc.
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "fmgr.h"
+
+#include "distributed/query_stats.h"
+
+PG_FUNCTION_INFO_V1(citus_stat_statements_reset);
+PG_FUNCTION_INFO_V1(citus_query_stats);
+
+
+/* placeholder for InitializeCitusQueryStats */
+void
+InitializeCitusQueryStats(void)
+{ }
```

Comment:
could maybe put a small comment in the function to fix the alignment and tell the reader why it is empty.